server/src/domain/media/media.util.ts

```diff
@@ -0,0 +1,199 @@
+import { BitrateDistribution, TranscodeOptions, VideoCodecSWHandler, VideoStreamInfo } from '.';
+import { SystemConfigFFmpegDto } from '..';
+
+abstract class BaseHandler {
```

Comment:
These don't really "handle" anything, they're just fancy classes to "get config". Can they be named `BaseConfig` and `VP9Config`, etc.?

Reply:
That's true. My thought was that they'd be handling different behavior for streams, but as it turns out they barely care about the stream contents.